django/db/models/query.py

```diff
@@ -494,6 +494,10 @@ def _extract_model_params(self, defaults, **kwargs):
             if f.attname in lookup:
                 lookup[f.name] = lookup.pop(f.attname)
         params = {k: v for k, v in kwargs.items() if LOOKUP_SEP not in k}
+        lookup_parameters = set(kwargs.keys()) - set(params.keys())
+        if not lookup_parameters_allowed and lookup_parameters:
+            raise TypeError("Lookups are not allowed for get_or_create. "
```

Comment:
get_or_create -> get_or_create(). The exception message can be much simpler in my opinion. Also,  if `len(lookup_parameters)` is 1, then the "Wrong parameters:" part is should be "Wrong parameter:" .